modules/xmpp/JingleSessionPC.js

```diff
@@ -713,7 +713,8 @@ export default class JingleSessionPC extends JingleSession {
                         error,
                         this.mediaConstraints);
                     finishedCallback(error);
-                });
+                },
+                this.mediaConstraints);
```

Comment:
This one looks like a "leftover bug" from "fix(JingleSessionPC): execute 'invite' on the queue" right? If so, can you please combine the commits?